versions/3.1.0.md

```diff
@@ -3264,6 +3264,7 @@ Field Name | Type | Description
 <a name="oauthFlowsPassword"></a>password| [OAuth Flow Object](#oauthFlowObject) | Configuration for the OAuth Resource Owner Password flow 
 <a name="oauthFlowsClientCredentials"></a>clientCredentials| [OAuth Flow Object](#oauthFlowObject) | Configuration for the OAuth Client Credentials flow.  Previously called `application` in OpenAPI 2.0.
 <a name="oauthFlowsAuthorizationCode"></a>authorizationCode| [OAuth Flow Object](#oauthFlowObject) | Configuration for the OAuth Authorization Code flow.  Previously called `accessCode` in OpenAPI 2.0.
+<a name="oauthFlowsDeviceCode"></a>deviceCode| [OAuth Flow Object](#oauthFlowObject) | Configuration for the OAuth Device Code flow.
```

Comment:
Before this gets merged, I have two questions I'd like to ask about this line.

1. I called it "Device Code" because that is how <https://oauth.net/2/grant-types/device-code> names it. However, the RFC is named "OAuth 2.0 Device Authorization Grant". Should I change this to be named "Device Authorization"?

2. Should [RFC8628](https://tools.ietf.org/html/rfc8628) be referenced here? I understand that the standard OAuth Flows don't have a RFC linked with them, but I feel like the Device Code flow may not be as well-known,

Reply:
See L:3164/5 for a paragraph which also needs to be updated, and would be a good place to add this link.

Reply:
Just to make sure I'm understanding that correctly -- you want me to put the RFC link there and *not* here? Or do you want me to put it both there and here?

Reply:
Just in the one place (the earlier paragraph) so that list of flows is expanded, and the link is present in the same part of the spec. as for other flows. Thanks.

Reply:
No problem.

Silly question though, should I refer to it as "OAuth2 device code flow", to keep it consistent with this name?

Or should I refer to it as "OAuth2 device authorization grant flow"? That's what the RFC calls it. If I refer to it as this, I might want to update the property name here too (see my other review, just below this, for more details on that).

Reply:
Though the only place we currently use the term "grant" in the spec is in the recommendation...

> Recommended for most use case is Authorization Code Grant flow with PKCE.

I'm inclined to go with what the RFC calls it.

Reply:
In that case, I think I'll use "OAuth2 device authorization flow", especially since it slides off the tongue a bit better too. I'll update this property name too, just to avoid confusion on which property is for this flow.